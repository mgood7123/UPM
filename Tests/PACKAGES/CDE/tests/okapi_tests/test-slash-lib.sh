#!/bin/sh

source okapi_test_common.sh
okapi_test_init # call init function

rm -f $TESTNAME.err.out

# ok, we're gonna try to copy ALL of /lib into $TESTDIR, which should be fun!
for f in `find /lib`
do
  $OKAPI_BIN $f "" $TESTDIR 2>> $TESTNAME.err.out
done

# also copy the entirety of /usr/src/kernels since something in lib references it
for f in `find /usr/src/kernels`
do
  $OKAPI_BIN $f "" $TESTDIR 2>> $TESTNAME.err.out
done

pushd $TESTDIR > /dev/null
find . | xargs file | sort > contents.txt
popd > /dev/null

diff -u $TESTDIR/contents.txt $TESTNAME.golden

diff -u $TESTNAME.err.out $TESTNAME.err.golden
