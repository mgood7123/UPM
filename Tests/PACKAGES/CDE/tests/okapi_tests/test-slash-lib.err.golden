WARNING: the symlink '/lib/modules/2.6.25.9-76.fc9.i686/source' has a non-existent target
WARNING: '/lib/modules/2.6.25.9-76.fc9.i686/../../../usr/src/kernels/2.6.25.9-76.fc9.i686' (symlink target) does not exist
WARNING: the symlink '/lib/modules/2.6.25.9-76.fc9.i686/build' has a non-existent target
WARNING: '/lib/modules/2.6.25.9-76.fc9.i686/../../../usr/src/kernels/2.6.25.9-76.fc9.i686' (symlink target) does not exist
